impeller/renderer/blit_pass.h

```diff
@@ -95,6 +120,13 @@ class BlitPass {
       IPoint destination_origin,
       std::string label) = 0;
 
+  virtual void OnCopyTextureToBufferCommand(
```

Comment:
Nit: I'd prefer if this returned bool so implementors could return error indications if they wished.

Reply:
Sounds good, done.